example/src/views/HomeView.vue

```diff
@@ -96,6 +158,21 @@ const icons = ref([
       <rui-icon v-for="(icon, i) in icons" :key="i" v-bind="icon" />
     </div>
   </div>
+
+  <div :class="css.section">
+    <h2 class="text-h4 mb-6" data-cy="checkboxes">Checkboxes</h2>
+    <div class="grid gap-1 grid-rows-2 grid-cols-6">
+      <rui-checkbox
+        v-for="(checkbox, i) in checkboxes"
+        :key="i"
+        v-bind="checkbox"
+        v-model:modelValue="checkbox.value"
```

Comment:
isn't this `v-model:modelValue=` == `v-model=` in vue 3? I seem to recall that we had this discussion at some point 😄

Reply:
Yeah, you're right. I thought I can't do that if I bind the v-model to 2 props.